### PR TITLE
command argument aliases that can be printed to help (DAT-17870)

### PR DESCRIPTION
**Pull request by StevenMassaro**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [ ] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 

## Description

Add ability for command argument aliases to be printed to the help output.

### CODE REVIEW

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java [hunk 1]
```diff
@@ -976,14 +977,17 @@ private void addSubcommand(CommandDefinition commandDefinition, CommandLine root
                     if (def.isRequired()) {
                         description = "[REQUIRED] " + description;
                     }
+                    if (!def.getForcePrintedAliases().isEmpty()) {
+                        String pluralized = def.getForcePrintedAliases().size() > 1 ? "aliases" : "alias";
+                        description = pluralized + ": '" + StringUtils.join(def.getForcePrintedAliases().stream().map(LiquibaseCommandLine::convertArgumentNameToKabobCase).collect(Collectors.toList()), "', '") + "'\n" + description;
```
Comment (abrackx) [inline liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java:982]: Can you use `Collectors.joining`?

**Review by abrackx — APPROVED** [review]
Nice, lgtm!

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java (hunk 1 above)
Comment (StevenMassaro) [reply liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java]: Good call

File: liquibase-standard/src/main/java/liquibase/command/CommandArgumentDefinition.java [hunk 2]
```diff
@@ -28,16 +27,66 @@ public class CommandArgumentDefinition<DataType> implements Comparable<CommandAr
...
+     *  of
+     *  but
+     *  can be explicitly with
+     * .
+     */
+    @Getter
     private String defaultValueDescription;
+    /**
+     * -- GETTER --
+     *  Function for converting values set in underlying
```
Comment (filipelautert) [inline liquibase-standard/src/main/java/liquibase/command/CommandArgumentDefinition.java:76]: Those auto-generated comments are broken - better to get rid or fix them. For instance, this one should be
```
    /**
     * Function for converting values set in underlying {@link liquibase.configuration.ConfigurationValueProvider}s into the
     * type needed for this command.
     */
```

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: liquibase-standard/src/main/java/liquibase/command/CommandArgumentDefinition.java (hunk 2 above)
Comment (StevenMassaro) [reply liquibase-standard/src/main/java/liquibase/command/CommandArgumentDefinition.java]: That's weird. I just assumed that the lombok plugin knew what it was doing. Nice catch. I just pushed up changes to the 3 that appear to be broken. I can also completely revert this if needed.

**Review by filipelautert — APPROVED** [review]

**Pull request merged**